logical_switch.go

```diff
@@ -271,3 +271,57 @@ func (odbi *ovnDBImp) lslblistImp(lswitch string) ([]*LoadBalancer, error) {
 	}
 	return listLB, nil
 }
+
+func (odbi *ovnDBImp) lsExtIdsAddImp(lswitch string, external_ids map[string]string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+	row := make(OVNRow)
+	row["name"] = lswitch
+	lsuuid := odbi.getRowUUID(tableLogicalSwitch, row)
+	if len(lsuuid) == 0 {
+		return nil, ErrorNotFound
+	}
+	if external_ids == nil {
+		return nil, fmt.Errorf("No external ids to set.")
+	}
+	mutateSet, err := libovsdb.NewOvsMap(external_ids)
```

Comment:
if the external_ids is empty, the err = nil, right?

Reply:
if empty, it means we raise error because we are trying to add/del empty ext_id maps

Reply:
So you mean we *SHOULD* raise error, but you didn't raise error because you are returning nil, nil at L284. Why?

	if len(external_ids) == 0 {
		return nil, nil
	}